### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by jeremyfrank**
Making some small tweaks to plugin docs.

### CODE REVIEW

File: examples/rect/README.md [hunk 1]
```diff
@@ -11,11 +11,9 @@ theme: {
     '24': ['24px', '24px'],
   },
 }
-variants: {
-  rect: ['responsive'],
-},
 plugins: [
-  require('@viget/tailwindcss-plugins/rect')(),
+  require('./config/tailwind/rect'),
```
Comment (olets) [inline examples/rect/README.md]: nit this assumes but doesn't explain the pattern of putting tailwind plugins in separate files within ./config/tailwind

File: examples/rect/README.md [hunk 2]
```diff
@@ -11,11 +11,9 @@ theme: {
     '24': ['24px', '24px'],
```
Comment (olets) [inline examples/rect/README.md:11]: while you're in here, this line is too indented

**Review by olets — APPROVED** [review]

**Review by nathan-schmidt-viget — APPROVED** [review]

File: examples/rect/README.md (hunk 1 above)
Comment (jeremyfrank) [reply examples/rect/README.md]: Any suggestions? How about `require('./path/to/your/plugin/rect'),`? We just need to show that it's not part of the bundled plugins.

File: examples/rect/README.md (hunk 2 above)
Comment (jeremyfrank) [reply examples/rect/README.md:11]: Oops will fix

Comment (jeremyfrank) [reply examples/rect/README.md:11]: Something must be up w/ Github previews. This is what I see:
image

**Comment by jeremyfrank** [comment]
Going to merge. If there's any additional feedback on that last commit, I can address in another PR.

**Pull request merged**